src/test/java/com/redhat/cloud/policies/app/UserConfigServiceTest.java

```diff
@@ -75,6 +81,18 @@ void passNotificationResponse() {
         }
     }
 
+    @Test
+    void testFedramp() {
+        when(environmentFlags.isFedramp()).thenReturn(Boolean.FALSE);
+
+        given()
+                .header(authHeader)
+                .when()
+                .get(PREFERENCE_URL)
+                .then()
+                .statusCode(404);
```

Comment:
Need to check why this is failing on travis but passing on my machine

Reply:
what was the reason for the failing test?

Reply:
The environment flag above was set to false, it should be true.